inginious/frontend/templates/custom_values.html

```diff
@@ -0,0 +1,90 @@
+{# This file is part of INGInious. See the LICENSE and the COPYRIGHTS files for #}
+{# more information about the licensing of this file. #}
+
+{% extends "layout.html" %}
+{% block title %}{{ course.get_name(user_manager.session_language()) }}{% endblock %}
+{% set registered = user_manager.course_is_user_registered(course) %}
+{% set staff = user_manager.has_staff_rights_on_course(course) %}
+{% block column %}
+    {{ template_helper.call('course_menu', course=course, template_helper=template_helper) | safe }}
+    {%if staff %}
+        {% if not course.is_open_to_non_staff() %}
+            <div class="alert alert-warning" role="alert">
+                {{ _("This course is currently invisible for students. You can change this by modifying the \"accessibility\" option in the configuration of the course.") }}
+            </div>
+        {% endif %}
+        <div class="list-group">
+            <a class="list-group-item list-group-item-action list-group-item-info" href="{{ get_homepath() }}/admin/{{ course.get_id() }}">
+                <i class="fa fa-user-secret fa-fw"></i>&nbsp; {{ _("Course administration") }}
+            </a>
+        </div>
+    {% elif registered %}
+        <div class="list-group">
+            <a class="list-group-item list-group-item-action list-group-item-info" href="{{ get_homepath() }}/group/{{ course.get_id() }}">
+                <i class="fa fa-group fa-fw"></i>&nbsp;
+                {% set mygroup = user_manager.get_course_user_group(course) %}
+                {% if mygroup and user_manager.session_username() in mygroup['students'] %}
+                    {{ _("Group : {}").format(mygroup['description']) }}
+                {% else %}
+                    {{ _("Group management") }}
+                {% endif %}
+            </a>
+            {% if course.allow_unregister() %}
+                <a href="#" data-toggle="modal" data-target="#unregister_modal" class="list-group-item list-group-item-action list-group-item-info">
+                    <i class="fa fa-user-times fa-fw"></i>&nbsp; {{ _("Unregister from this course") }}
+                </a>
+            {% endif %}
+            <a class="list-group-item list-group-item-action list-group-item-info" href="{{ get_homepath() }}/custom/{{ course.get_id() }}">
+                <i class="fa fa-cogs fa-fw"></i>&nbsp; {{ _("Custom fields") }}
+            </a>
+        </div>
+    {% elif not user_manager.session_logged_in() %}
```

Comment:
This page is not meant to be accessible to anonymous users as it is a user settings page... This is probably dead code as the page class inherits from ``INGIniousAuthPage``.